rac-proxy/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "rac-proxy"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+anyhow = "1.0.86"
+clap = { version = "4.5.16", features = ["derive"] }
+env_logger = "0.11.5"
+hex = "0.4.3"
+openssl = "0.10.66"
+raclients = { path = "../raclients", features = ["frontend"] }
```

Comment:
Why we need to enable "frontend" in the proxy?

Reply:
The proxy serves as frontend server and backend client. raclients::frontend::FrontendServer is required by the proxy.